src/js/header.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './css/header.css';
+
+const Header = () => (
+  <header>
+    <h1>Edith Magic</h1>
+    <ul>
+      <a href="/">Home</a>
+      <a href="/Calculator">Calculator</a>
```

Comment:
Great job @vigehi 🤝

- [ ] Kindly take a closer look at the screen of your calculator when you perform a mathematical calculation. The entries seem to be hidden and then the final result is displayed. Kindly make sure the transactions are displayed before the final result is displayed.

![calculator](https://user-images.githubusercontent.com/63917725/191080528-143a6ac9-cd12-42f6-9930-ef92a9c12e77.PNG)

Reply:
Well resolved 👍🏽